metabase/metabase
# 2. Add the rest 3 illustration settings

Epic: https://github.com/metabase/metabase/issues/39246

### Description

This PR add all the necessary BE settings to the project and connect it to the new UI, but those new 3 settings won't be used yet.

### How to verify

1. Go to /admin/settings/whitelabel/conceal-metabase.
2. Play around with the illustration settings. Make sure the change persists.
3. The setting widget shouldn't behave any differently than before, though the design will be updated to match the new design in the next PR.

### Demo

<img width="740" alt="image" src="https://github.com/metabase/metabase/assets/1937582/17ebada3-4f59-4f40-bbcb-d09d2ec22f5b">

### Checklist

- [ ] ~Tests have been added/updated to cover changes in this PR~ Tests will be added in a subsequent PR.
